deepchem/models/tests/test_losses.py

```diff
@@ -0,0 +1,199 @@
+import deepchem.models.losses as losses
+import unittest
+import numpy as np
+
+try:
+  import tensorflow as tf
+  has_tensorflow = True
+except:
+  has_tensorflow = False
+
+try:
+  import torch
+  has_pytorch = True
+except:
+  has_pytorch = False
+
+
+class TestLosses(unittest.TestCase):
```

Comment:
It might be good to convert these tests to the new pytest style (functions without the wrapper `unittest.TestCase` class